map_gui/src/tools/importer.rs

```diff
@@ -96,7 +112,7 @@ impl<A: AppLike + 'static> State<A> for ImportCity<A> {
                         format!("{}/one_step_import", bin_dir),
                         "boundary.geojson".to_string(),
                     ];
-                    if !self.panel.is_checked("driving side") {
+                    if self.panel.is_checked("drive on the left") {
```

Comment:
I inverted this so that the "left" value could be on the left of the switch. I haven't been able to test it because the importer is still running...

Reply:
I've since tested this*, and everything looks good. Fun to see people in Utah driving on the left side of the road.

*on regions with less data than souther california, so the import could complete